src/index.js

```diff
@@ -4,15 +4,20 @@ import './index.scss';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-import { BrowserRouter as Router } from 'react-router-dom'; 
+import _reducer from './reducers';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import { applyMiddleware, createStore, combineReducers } from 'redux';
 
-const output = (
-  <Router>
-    <App />
-  </Router>
-)
+const reducer = combineReducers({ _reducer });
```

Comment:
so I think Frank was using itemReducer here.  After we merge we'll just have to make sure we we get all of our reducers combined here appropriately